src/main/java/org/betonquest/betonquest/events/ObjectiveEvent.java

```diff
@@ -24,39 +25,55 @@
 @CustomLog
 public class ObjectiveEvent extends QuestEvent {
 
-    private final ObjectiveID objective;
+    /**
+     * The BetonQuest instance.
+     */
+    public static BetonQuest BETONQUEST;
```

Comment:
Now there is no reason to keep this field static anymore.
```suggestion
    private final BetonQuest betonQuest;
```
or if other classes might extend this class (which should not be the case)
```suggestion
    protected final BetonQuest betonQuest;
```